templates/KnockoutSpa/ClientApp/components/app-root/app-root.ts

```diff
@@ -40,4 +40,4 @@ class AppRootViewModel {
     }
 }
 
-export default { viewModel: AppRootViewModel, template: require('./app-root.html') };
+export default { viewModel: AppRootViewModel, componentName: 'Home', template: require('./app-root.html') };
```

Comment:
So when a refresh is done it doesn't load that routed content, but when the link is clicked, it works fine. Any idea what could be causing this?